fastlane/Fastfile

```diff
@@ -18,6 +18,7 @@
     releaseNotesBodyHeader = "What's new:"
     releaseNotesFileBody = "../app/version/release-notes"
     appVersionFilePath = "../app/version/version-properties"
+    firebaseFilePath = "#{ENV["HOME"]}/jenkins_static/com.duckduckgo.mobile.android/ddg-upload-firebase.json"
```

Comment:
Can’t use the ENV from outside the fastfile. We can leave it as is now and come back at it later. I’ve tried passing the value in different ways but wasn’t successful.